src/helper.jl

```diff
@@ -165,3 +165,167 @@ function writetable(filename::AbstractString; rewrite::Bool=false, kw...)
     writexlsx(filename, xf, rewrite=rewrite)
     nothing
 end
+
+"""
+    emptyfile(sheetname::AbstractString="")
+
+Returns an empty, writable `XLSXFile` with 1 worksheet.
+
+`sheetname` is the name of the worksheet, defaults to `Sheet1`.
+"""
+function emptyfile(sheetname::AbstractString="")
+    xf = open_default_template()
+
+    if sheetname != ""
+        rename!(xf[1], sheetname)
+    end
+
+    return xf
+end
```

Comment:
Instead of doing (or in addition) do:
```julia
xf = XLSX.emptyfile(sheetname)
XLSX.writerow(xf, ...)
XLSX.closefile(xf, filename)
```
You could instead define:
```julia
function Base.open(f::Base.Callable, filename::AbstractString, sheetname::AbstractString="")
    xf = open_default_template()

    if sheetname != ""
        rename!(xf[1], sheetname)
    end

    try
        f(xf)
    finally
        XLSX.closefile(xf, filename)
    end
end
```
and then it could be utilized via:
```julia
open(filename, sheetname) do xf
    XLSX.writerow(xf, ...)
end
```